src/main/java/org/arl/fjage/remote/MasterContainer.java

```diff
@@ -384,9 +385,9 @@ public void shutdown() {
       slaves.clear();
       needsCleanup = false;
     }
-    if (listener != null) {
-      listener.close();
-      listener = null;
+    if (tcpListener != null) {
+      tcpListener.close();
+      tcpListener = null;
     }
```

Comment:
We should close the `websocketListener` also, no?

Reply:
Good catch!